.github/workflows/ci.yml

```diff
@@ -90,7 +94,7 @@ jobs:
         uses: actions-rs/cargo@v1.0.3
         with:
           command: clippy
-          args: --all-features --all-targets -- -D warnings
+          args: --all-features --all-targets -- -A clippy::uninlined_format_args ${{ matrix.args }}
```

Comment:
Similar to s2n-quic https://github.com/aws/s2n-quic/blob/main/.github/workflows/ci.yml#L98